content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -176,6 +176,59 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using automated rules in Odoo. Odoo *Studio* is required to use
+   automated rules. To install it, go to :menuselection:`Apps --> Search...` bar and search for
+   `studio`. If it is not already installed, click :guilabel:`Install`. Adding the *Studio*
+   application upgrades the subscription status to *Custom*, which increases the cost. Consult
+   `support <https://www.odoo.com/contactus>`_ or reach out to the database's customers success
+   manager with any questions on upgrading.
+
+   To use automated rules, navigate to :menuselection:`Settings app --> Technical menu -->
+   Automation section --> Automated Rules`. Then, click :guilabel:`New` to create a new rule.
+
+   Enter a name for the automated rule and select a :guilabel:`Model` to implement this rule on.
+   Once setting the :guilabel:`Model` other options will populate.
+
+   Set the :guilabel:`Trigger` to one of the following options:
+
+   :guilabel:`Value Updated`
+    - :guilabel:`User is set`
+    - :guilabel:`State is set to`
+    - :guilabel:`On archived`
+    - :guilabel:`On unarchived`
+
+   :guilabel:`Timing Conditions`
+    - :guilabel:`Based on date field`
+    - :guilabel:`After creation`
+    - :guilabel:`After last update`
+
+   :guilabel:`Custom`
+    - :guilabel:`On save`
+    - :guilabel:`On deletion`
+    - :guilabel:`On UI change`
+
+   :guilabel:`External`
+    - :guilabel:`On webhook`
+
+   Based on the selection for the :guilabel:`Trigger`, additional fields will populate.
+
+   Under the :guilabel:`Before Update Domain`, set a condition to be met before updating the record.
+   Click :guilabel:`Edit Domain` to set record parameters.
+
+   Under the :guilabel:`Actions To Do` tab, select :guilabel:`Add an action`. Next, select
+   :guilabel:`Send SMS`, and set the :guilabel:`SMS Template`. Choose whether the SMS message should
```

Comment:
```suggestion
   :guilabel:`Send SMS`. Then, set the :guilabel:`SMS Template`. Choose whether the SMS message should
```
is there a reason the 'Technical Settings' section was bypassed during this explanation? Do you want to consider adding it in during your revision? Just a quick explanation should work...